docs/configuration.asciidoc

```diff
@@ -148,6 +148,27 @@ NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-service-node-name]]
+==== `ServiceNodeName` (added[1.2])
```

Comment:
@gregkalapos, I believe we need to change version here, as this feature isn't included in 1.2 version. Maybe, we need to think about some placeholders which will be replaced during release to avoid issues with documentation

Reply:
Thanks for tagging me @vhatsura! Opened https://github.com/elastic/apm-agent-dotnet/pull/627 - yeah we should came up with something for this.